mixins/localize-mixin.js

```diff
@@ -54,35 +43,32 @@ export const LocalizeMixin = superclass => class extends superclass {
 
 				// Everytime language or resources change, invalidate the messages cache.
 				const proto = this.constructor.prototype;
-				this.checkLocalizationCache(proto);
+				this._initLocalizationCache(proto);
 				proto.__localizationCache.messages = {};
 
-				this.getLangResources(this.__language)
-					.then((res) => {
-						if (!res) {
-							return;
-						}
-						this._onRequestResponse(res, this.__language);
-					});
+				this.__namespace = `${this.__namespaceBase}:${this.__language}`;
```

Comment:
Shouldn't this be `this.namespaceBase` (without the double underscores)?

Reply:
it definitely should